variable-level-metadata-schema/docs/md-rendered-schemas/jsonschema-jsontemplate-data-dictionary.md

```diff
@@ -350,57 +365,104 @@ a physical string representation to be cast as false (increasing
 readability of the field) that is not a standard false value. It can include one or more values.
 
 
-**`repo_link`** _(string)_
- A link to the variable as it exists on the home repository, if applicable
-
-
 **`standardsMappings`** _(array)_
- A published set of standard variables such as the NIH Common Data Elements program.
-
-**`relatedConcepts`** _(array)_
- Mappings to a published set of concepts related to the given field such as ontological information (eg., NCI thesaurus, bioportal etc)
-
-**`univarStats`** _(object)_
- Univariate statistics inferred from the data about the given variable 
-
-
-
-- **`median`** _(number)_
-     
-
-
-- **`mean`** _(number)_
-     
-
+ 
+A set of instrument and item references to standardized data elements designed to document
+the [HEAL common data elements program](https://heal.nih.gov/data/common-data-elements)
+and other standardized/common element sources to facilitate cross-study comparison and interoperability
+of data. One can either map an individual data element or an instrument in which the field is 
+a part of.
 
-- **`std`** _(number)_
-     
+__**All Fields Mapped (Both Instrument and Item)**__
 
+```json
+"standardsMappings": [
+    {
+        "instrument": {
+            "url": "https://www.heal.nih.gov/files/CDEs/2023-05/adult-demographics-cdes.xlsx",
+            "source": "heal-cde",
+            "title": "adult-demographics",
+            "id": <drupal id here>
```

Comment:
I think Markdown is going to be unhappy with angled brackets in here, since Markdown allows HTML tags, and that's what this is going to look like. Maybe:

```suggestion
            "id": "[Drupal ID here]"
```

Reply:
Thanks! Didn't realize I left this placeholder in here. It looks like these ids are just in the form of 4 digits so included an instance of instead of palceholer